Expensify/App
# fix styling for report typing and timezone indicators

### Details

1. Reduced gap between chat list and composer.
2. Removed boldness from the typing indicator & Timezone info.
3. Added Wrapping to both of the above.

### Fixed Issues

$ https://github.com/Expensify/App/issues/3987

### Tests | QA Steps
1. Open NewDot on Device A.
2. Open a chat with a user at a different timezone. Check the Timezone info.
3. Type a message on Device B & and check the typing status on Device A.

### Tested On


### Screenshots


#### Web

![web-com-gap](https://user-images.githubusercontent.com/24370807/127401509-d240b010-3b4f-4e65-b7c5-6bd6e7093e83.png)
![web-com](https://user-images.githubusercontent.com/24370807/127401521-34d80c5f-692d-40fe-bfe5-ab9d50f9f86c.png)

#### Mobile Web

![com-m](https://user-images.githubusercontent.com/24370807/127403068-1def05fc-5d1d-4d92-9526-75223c9bf155.png)

#### Desktop

![com-d](https://user-images.githubusercontent.com/24370807/127406863-45d70a5a-a461-4691-8afc-36e52da97f60.png)

#### iOS

![com-i](https://user-images.githubusercontent.com/24370807/127407552-efe8e82c-22a3-446e-aafa-f60b5c8fe292.png)

#### Android

![com-a](https://user-images.githubusercontent.com/24370807/127402522-e1d10f89-acb2-4f07-9229-1b68e97227d9.png)
